File: app/src/main/java/vn/loitp/app/activity/customviews/layout/draggablepanel/FrmTestBottom.java
package vn.loitp.app.activity.customviews.layout.draggablepanel;

/**
 * Created by www.muathu@gmail.com on 12/24/2017.
 */

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import loitp.basemaster.R;

public class FrmTestBottom extends Fragment {
    public static FrmTestBottom newInstance() {
        return new FrmTestBottom();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.frm_test, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        ImageView iv = (ImageView) view.findViewById(R.id.iv);
        iv.setImageResource(R.drawable.iv);
    }
}

File: app/src/main/java/vn/loitp/app/a/customviews/layout/draggablePanel/FrmTestBottom.kt
package vn.loitp.app.a.customviews.layout.draggablePanel

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import kotlinx.android.synthetic.main.frm_test_bottom.*
import vn.loitp.R

class FrmTestBottom : Fragment() {

    companion object {
        fun newInstance(): FrmTestBottom {
            return FrmTestBottom()
        }
    }

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.frm_test_bottom, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        imageView.setImageResource(R.drawable.iv)
    }
}
